media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.600.2900.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit unpacker
+
+MY_PV="$(ver_cut 1-3)"
+
+DESCRIPTION="HDX RealTime Media Engine plugin for Citrix Workspace App"
+HOMEPAGE="https://www.citrix.com/"
+SRC_URI="amd64? ( HDX_RealTime_Media_Engine_${MY_PV}_for_Linux_x64.zip )
+	x86? ( HDX_RealTime_Media_Engine_${MY_PV}_for_Linux.zip )"
+LICENSE="icaclient"
+SLOT="0"
+
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="fetch mirror strip"
+
+BDEPEND="app-arch/unzip"
+RDEPEND="
+	sys-libs/llvm-libunwind
+	|| (
+		media-video/pipewire
+		media-sound/pulseaudio
+		media-sound/apulse
+	)
+	!<net-misc/icaclient-22.12.0.12
+"
```

Comment:
```
 * QA Notice: Unresolved soname dependencies:
 *
 *  /opt/Citrix/ICAClient/rtme/RTMediaEngineSRV: libX11.so.6 libXv.so.1 libpulse.so.0 libsndfile.so.1
 *
```
Please add the providers for these libs into RDEPEND.

Reply:
I added the libs for the first two, the later two i put on a ignore list like in the parent ebuild

this thing just has multiple implementations and ldd shows stuff that might not all be used/needed